cluster/gce/util.sh

```diff
@@ -964,10 +966,15 @@ function kube-down {
         --project "${PROJECT}" \
         --quiet \
         --zone "${ZONE}" \
-        "${group}"
+        "${group}" &
```

Comment:
Actually, if there would be a lot of MIGs, this may result in too many migs being removed concurrently. So maybe drop a TODO to change it in the future.